app/models/wiki_page.rb

```diff
@@ -189,12 +189,12 @@ def merge_version
 
   def merge_version?
     prev = versions.last
-    prev && prev.updater_id == CurrentUser.user.id && prev.updated_at > 1.hour.ago
+    prev && prev.updater_id == CurrentUser.id && prev.updated_at > 1.hour.ago
   end
 
   def create_new_version
     versions.create(
-      :updater_id => CurrentUser.user.id,
+      :updater_id => CurrentUser.id,
```

Comment:
This seems like a bandaid fix. `CurrentUser.id` will return `nil` if `CurrentUser` isn't set, which is still wrong.

Reply:
Yeah, it was mostly just to get the tests to work.